content/developer/writing_widgets/widget-definition.md

```diff
@@ -140,8 +140,13 @@ fetchUrl: '[manager]/executions?is_system_workflow=false[params]'
     * `[params:param_name1(,param_name2)]` is replaced with "&paramName1:paramValue1" in the URL.
      Please note that this can be used both to selectively pick pagination parameter as well as custom parameters (see [fetchParams function](#fetchparams-widget-toolbox)).
       This mode is **exclusive** - parameters not specified explicitly are skipped.
-      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include all pagination parameters (`_size`, `_offset`, `_sort`) instead of specifying explicitly  each of the three.
-
+      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include [pagination parameters](https://docs.cloudify.co/api/v3.1/#pagination) (`_size`, `_offset`), [sorting parameter](https://docs.cloudify.co/api/v3.1/#sorting) (`_sort`) and search parameter (`_search`) instead of specifying explicitly each of the four. It means `fetchUrl: '[manager]/agents?[params:gridParams]'` is equivalent for 
+      `fetchUrl: '[manager]/agents?[params:_offset,_size,_sort,_search]'`.
+      
+      Useful resources: 
+        * [`DataTable` documentation](https://docs.cloudify.co/ui-components/2.2.0/?path=/docs/home--page),
```

Comment:
Using a hardcoded version has a quick potential to get out of date :worried: 

Could we somehow set the version once, and reuse it across the docs?

We already specify the version in https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/b31c6a78ada9037bf834b24a6e12fae7db5f4418/content/developer/writing_widgets/widgets-components.md#L9

Reply:
fixed